external/storm-elasticsearch/src/test/java/org/apache/storm/elasticsearch/common/EsTestUtil.java

```diff
@@ -46,6 +47,30 @@ public Fields getComponentOutputFields(String componentId, String streamId) {
         return new TupleImpl(topologyContext, new Values(source, index, type, id), 1, "");
     }
 
+    public static EsTupleMapper generateDefaultTupleMapper() {
+        return new EsTupleMapper() {
+            @Override
+            public String getSource(ITuple tuple) {
+                return tuple.getStringByField("source");
+            }
+
+            @Override
+            public String getIndex(ITuple tuple) {
+                return tuple.getStringByField("index");
+            }
+
+            @Override
+            public String getType(ITuple tuple) {
+                return tuple.getStringByField("type");
+            }
+
+            @Override
+            public String getId(ITuple tuple) {
+                return tuple.getStringByField("id");
+            }
+        };
+    }
+
```

Comment:
@sweetest 
SampleEsTupleMapper is just a class I defined from README.
But I also think it would be better to have default mapper, and its name would be DefaultEsTupleMapper. 
I'll reflect it. Thanks!